packages/i18n/package.json

```diff
@@ -18,13 +18,20 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./index.js",
-  "module": "./index.js",
+  "main": "dist/i18n.cjs.js",
+  "module": "dist/i18n.es.js",
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "cross-env npm run build:es && npm run build:cjs",
+    "build:watch": "cross-env npm run build:es -- -w",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs index.js -o dist/i18n.cjs.js",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es index.js -o dist/i18n.es.js"
+  },
   "dependencies": {
-    "@commercetools-frontend/i18n": "1.0.0-rc.3",
-    "@commercetools-frontend/sentry": "1.0.0-rc.3",
+    "@commercetools-frontend/sentry": "1.0.0-rc.2",
     "moment": "2.22.2",
     "prop-types": "15.6.2",
-    "react": "16.6.0"
+    "react": "16.6.0",
+    "react-intl": "2.7.1"
```

Comment:
Uh, those should be defined as peer dependencies. I guess I missed that last time #34 🙈